### PR TITLE
Command to send 10ADS to self

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

File: app/Http/Controllers/SupplyController.php [hunk 1]
```diff
@@ -242,9 +243,11 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
         $url = Utils::addUrlParameter($url, 'k', Utils::urlSafeBase64Encode(json_encode($log->our_userdata)));
         $url = Utils::addUrlParameter($url, 'r', Utils::urlSafeBase64Encode($backUrl));
 
-        $response = new RedirectResponse($url);
+        return new Response(null, Response::HTTP_NO_CONTENT);
 
-        return $response;
+//        $response = new RedirectResponse($url);
```
Comment (adshares-tech) [inline app/Http/Controllers/SupplyController.php:248]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Http/Controllers/SupplyController.php [hunk 2]
```diff
@@ -242,9 +243,11 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
...
+//
+//        return $response;
```
Comment (adshares-tech) [inline app/Http/Controllers/SupplyController.php:250]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

**Comment by adshares-tech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: app/Console/Commands/AdsSend.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+class AdsSend extends Command
+{
+    protected $signature = 'ads:send';
+
+    public function handle(AdsClient $adsClient)
+    {
+        $response = $adsClient->runTransaction(new SendOneCommand(
+            config('app.adshares_address'),
+            10 * 10
+            ^ 11, '0000000000000000000000000000000028a9dbfdb3244297b0e1bb66fc0dceb8'));
```
Comment (PawelPodkalicki) [inline app/Console/Commands/AdsSend.php]: change ^ with **
```suggestion
            10 * 10 ** 11,
```

**Review by PawelPodkalicki — COMMENTED** [review]
Operator for power is **, not ^

**Review by PawelPodkalicki — APPROVED** [review]

**Review by PawelPodkalicki — CHANGES_REQUESTED** [review]
Operator for power is **, not ^

**Review by PawelPodkalicki — APPROVED** [review]

**Pull request merged**